packages/grafana-ui/src/components/Button/Button.mdx

```diff
@@ -43,7 +43,7 @@ The secondary button, used for "cancel" or aborting.
 
 ## Destructive
 
-Used for removing or deleting entities.
+Used for "call to action", i.e. triggering the main action, when that main action is removing or deleting something. Because of its dominant coloring, it should be used sparingly. If you need multiple Destrucutive `Button` components in one view, maybe use a secondary `Button` or Link variant instead and only use the Destructive variant to double confirm. Just like the primary variant, the Destructive `Button` can be juxtaposed with secondary buttons like "Cancel" or "Abort".
```

Comment:
1. "call to action" isn't usually associated with destructive actions by people. Maybe we could simplify and say:
"used for triggering action which is removing or deleting something."
2. "maybe use a secondary `Button`..." –> "we recommend using a secondary `Button`..."
3. Not sure the last sentence is needed here.